collector/util.go

```diff
@@ -16,10 +16,26 @@ package collector
 
 import "github.com/google/cadvisor/container"
 
-func (endpointConfig *EndpointConfig) configure(containerHandler container.ContainerHandler) {
+var DefaultURLConfig = URLConfig{
```

Comment:
I don't think this is a good idea. If defaults exist, they should be per collector and not generic. For example, mysql metrics might be available at a well known socket. nginx metrics need not be available there.